memfault-ecs-cli/meta.yaml

```diff
@@ -0,0 +1,22 @@
+{% set name = "ecs-cli" %}
+{% set version = "1.21.0.mflt" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/memfault/memfault-ecs-cli/releases/download/{{ version }}/ecs-cli.darwin-amd64 # [osx]
+  sha256: 13c631695ada3d14ecdeb6ea4896ab4e94bf9e3a6f574b2ed57596643f0db496 # [osx]
+  url: https://github.com/memfault/memfault-ecs-cli/releases/download/{{ version }}/ecs-cli.linux-amd64 # [linux64]
+  sha256: 80f7eee721e1b0268173c8a5a1400486ddae409d737b6c0f4b5fd58baa37e2bb # [linux64]
```

Comment:
for some reason i thought we'd be building via `go` within this recipe but instead we just pull down the pre-built binary. neat! simple!

i could have seen it either way but this one seems fine

Reply:
I think that's an option we could've gone w/ but AWS packed the original repo w/ a makefile that I've been using to do the builds!

There'd probably be a way to pull that apart and do the proper invocations but as a certified non-expert at go, I actively tried to avoid it in the short term. If we ever needed to get much better at maintaining this across more platforms, I suspect we'd have to go that way.